apps/playground-web/src/components/styled-connect-button.tsx

```diff
@@ -18,6 +18,7 @@ export function StyledConnectButton(
         "84532": ["0x638263e3eAa3917a53630e61B1fBa685308024fa"],
       }}
       client={THIRDWEB_CLIENT}
+      locale="kr_KR"
```

Comment:
leftover from testing? - maybe we should have a language selector in the playground to let people try out different locales?

Reply:
oops yeah let me remove